### PR TITLE
Release v1.1.1

### PR DESCRIPTION
**Pull request by phpfs**
Release notes:
- Include information on SIGMOD 2021
- Minor bugfixes

### CODE REVIEW

File: docs/sigmod2021.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+calculate binary metrics like precision or recall.
+
+We are working closely with the contest team to allow for a seamless experience.
+
+## Setup
+
+To use snowman, you'll have to download the latest [release (Github Releases)](https://github.com/HPI-Information-Systems/snowman/releases) of
+snowman for your specific os. After that you are able to start snowman!  
+As a first step, we'd suggest for you to start with our section [Basic Usage](/basic_usage/introduction)
+for a guide of how to benchmark and evaluate with snowman.
```
Comment (martingraf4) [inline docs/sigmod2021.md:21]: evaluate what?

File: docs/sigmod2021.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Some contest datasets are already bundled with the release. Select on of them, upload your own experiment result and
+start benchmarking!
```
Comment (martingraf4) [inline docs/sigmod2021.md:24]:
```suggestion
Some contest datasets are already bundled with the release. Select one of them, upload your own experiment result and
start benchmarking!
```

File: docs/sigmod2021.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+In case you'll want to upgrade later on, simply download the newest release and you're ready to go!
+
+## Datasets
+
+We've included an automatic importer for the gold standard format and validated that the below datasets can be imported
+out of the box. More datasets will be added as soon as they were released by the contest team.
```
Comment (martingraf4) [inline docs/sigmod2021.md:31]:
```suggestion
We've included an automatic importer for the gold standard format and validated that the below datasets can be imported
out of the box. More datasets will be added as soon as they are released by the contest team.
```

**Review by martingraf4 — CHANGES_REQUESTED** [review]

File: docs/sigmod2021.md (hunk 1 above)
Comment (phpfs) [reply docs/sigmod2021.md:21]: Well - the algorithm the user is working with ;) Will change this :)

File: docs/sigmod2021.md (hunk 2 above)
Comment (phpfs) [reply docs/sigmod2021.md:24]: ok

File: docs/sigmod2021.md (hunk 3 above)
Comment (phpfs) [reply docs/sigmod2021.md:31]: ok

**Review by martingraf4 — APPROVED** [review]

**Pull request merged**